interview-questions.md

```diff
@@ -1,4 +1,4 @@
-# ASSESSMENT 1: Tech Interview Practice Questions
+<!-- # ASSESSMENT 1: Tech Interview Practice Questions
```

Comment:
You don't need to commented out anything on this file.

Reply:
Duly noted in the future, thank you and roger that  :)